pantheon-bundle/frontend/src/app/contentDisplay.tsx

```diff
@@ -109,15 +111,32 @@ class ContentDisplay extends Component<any, IModuleDisplayState | IAssemblyDispl
                         {this.state.lastPublishDate.trim() !== "" && this.state.lastPublishDate !== "-"
                             && this.state.variantUUID !== ""
                             && this.state.portalUrl !== ""
+                            && this.state.portalUrlType === "LIVE"
                             && <span><a href={this.state.portalUrl} target="_blank">View on Customer Portal  <i className="fa pf-icon-arrow" /></a> </span>
                         }
+                        {this.state.variantUUID !== ""
+                            && this.state.portalUrl !== ""
+                            && this.state.portalUrlType === "PRELIVE"
+                            && <span><a href={this.state.portalUrl} target="_blank">Pre-live Customer Portal URL</a> </span>
+                        }
                     </LevelItem>
                     <LevelItem>
                         {this.state.lastPublishDate.trim() !== "" && this.state.lastPublishDate !== "-"
                             && this.state.variantUUID !== ""
                             && this.state.portalUrl !== ""
+                            && this.state.portalUrlType === "LIVE"
                             && <span><a id="permanentURL" onClick={this.copyToClipboard} onMouseLeave={this.mouseLeave}>Copy permanent URL  <CopyIcon /></a></span>
                         }
+                        {this.state.variantUUID !== ""
+                            && this.state.portalUrl !== ""
+                            && this.state.portalUrlType === "PRELIVE"
+                            && <span><a id="permanentURL" onClick={this.copyToClipboard} onMouseLeave={this.mouseLeave}>Copy pre-live URL  <CopyIcon /></a></span>
```

Comment:
Does the pre-live URL look different from the permanent URL?

Reply:
when there are more than one urls to copy, i think we are supposed to use a dropdown:
https://marvelapp.com/prototype/70e251a/screen/67615882
https://www.patternfly.org/v4/components/dropdown

Reply:
The pre-live URL would likely not look any different. The only difference is the wording of the label shown through the UI, to remind authors that the URL would change if they change the product metadata.

We don't need to use a dropdown because there's only one URL to copy. The code is surrounded by conditionals and only one conditional will evaluate to true at a time.

I think that addresses your concerns, but let me know if there's something I missed.

Reply:
if only one url is displayed, then I'm fine without the dropdown.